src/controllers/controllerengine.h

```diff
@@ -107,6 +107,7 @@ class ControllerEngine : public QObject {
     Q_INVOKABLE void scratchDisable(int deck, bool ramp = true);
     Q_INVOKABLE bool isScratching(int deck);
     Q_INVOKABLE void softTakeover(QString group, QString name, bool set);
+    Q_INVOKABLE void softTakeoverResync(QString group, QString name);
```

Comment:
Reset is in the same way confusing as Resync since the scope is fuzzy. Lets stick with Resync or long name that describes exactly what it does or when it should be called. 
Something like: 

softTakeoverFromAnOutOfSyncPot()